### PR TITLE
Create note deletion UI

### LINKED ISSUE #8: Create note deletion UI

### PR DESCRIPTION
**Pull request by chiragbagde**

### CODE REVIEW

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 1]
```diff
@@ -1,38 +1,23 @@
...
 
   return (
     <Alert
       isOpen
       isSubmitting={deleting}
-      message="Are you sure you want to continue? This cannot be undone."
-      title={`Delete ${selectedNoteIds.length} ${
-        selectedNoteIds.length > 1 ? "notes" : "note"
-      }?`}
+      message={`Are you sure you want to continue deleting ${title} This cannot be undone.`}
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:19]: Add proper punctuations in sentence.

File: app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx [hunk 2]
```diff
@@ -1,38 +1,23 @@
...
+      title="Delete the note?"
```
Comment (yedhink) [inline app/javascript/src/components/Dashboard/Notes/DeleteAlert.jsx:20]: Keep it as `Delete note?`

**Review by yedhink — COMMENTED** [review]
@chiragbagde

**Comment by chiragbagde** [comment]
I've done the changes.

**Pull request merged**